<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1089</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>实验13 视窗变换</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_ww">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>72</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>窗口宽度：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_wh">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>72</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>窗口高度：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_vw">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>20</y>
      <width>72</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>视口宽度：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_vh">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>20</y>
      <width>72</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>视口高度：</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_ww">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_wh">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_vw">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>20</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_vh">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>20</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1089</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
